### PR TITLE
Requests are being blocked because their originating IP is undefined

### PR DESCRIPTION
**Pull request by terovirtanen**
BEGIN_COMMIT_OVERRIDE
fix: set X-Forwarded-For only if clientIP exists [PLATTA-4776](https://helsinkisolutionoffice.atlassian.net/browse/PLATTA-4776)
fix: upgrade to node14 [YM-517](https://helsinkisolutionoffice.atlassian.net/browse/YM-517)
END_COMMIT_OVERRIDE

[PLATTA-4776]: https://helsinkisolutionoffice.atlassian.net/browse/PLATTA-4776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[YM-517]: https://helsinkisolutionoffice.atlassian.net/browse/YM-517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**youth-federation-gateway branch is deployed to platta: https://youth-federation-gateway-pr19.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**youth-federation-gateway branch is deployed to platta: https://youth-federation-gateway-pr19.dev.hel.ninja** :rocket::rocket::rocket:

File: src/dataSources.ts [hunk 1]
```diff
@@ -183,9 +183,11 @@ export class AuthenticatedDataSource extends FileUploadDataSource {
           (context as any).acceptLanguage
       );
 
-      request.http.headers.set(
-          "X-Forwarded-For",
-          (context as any).clientIP
-      );
+      if ((context as any).clientIP) {
```
Comment (nikomakela) [inline src/dataSources.ts]:
```suggestion
      if ((context as any)?.clientIP) {
```

**Review by nikomakela — APPROVED** [review]
I would add 1 question mark, since the any can also be null.

**Comment by terovirtanen** [comment]
**youth-federation-gateway branch is deployed to platta: https://youth-federation-gateway-pr19.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**